src/libraries/Microsoft.Extensions.FileProviders.Physical/src/Microsoft.Extensions.FileProviders.Physical.csproj

```diff
@@ -2,11 +2,17 @@
 
   <PropertyGroup>
     <RootNamespace>Microsoft.Extensions.FileProviders</RootNamespace>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461;$(NetFrameworkCurrent)</TargetFrameworks>
+    <ExcludeCurrentFullFrameworkFromPackage>true</ExcludeCurrentFullFrameworkFromPackage>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultItems>true</EnableDefaultItems>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Ensure Assemblies are first resolved via targeting pack when targeting net461 -->
+    <AssemblySearchPaths Condition="'$(TargetFramework)' == 'net461'">$(NuGetPackageRoot)\microsoft.targetingpack.netframework.v4.6.1\1.0.1\lib\net461\;$(AssemblySearchPaths)</AssemblySearchPaths>
```

Comment:
Would it make sense to just stop binplacing `NETStandardLibrary.NETFramework` into RefPath completely?  That's only there to support loading/running .NETStandard assemblies on .NETCore, and with this change don't we aim to eliminate all the cases where we would need to do that?

Reply:
Or maybe this is just another opportunity for @ViktorHofer to clean up in his PR that eliminates the RefPath for netfx.

Reply:
I'm pretty sure I tried doing exactly this on release 3.1 branch when I tried to fix this error initially and found many build breaks because of it, and I tried working through the errors but gave up since it felt like peeling an onion and I found the workaround for using AssemblySearchPaths instead. I'm happy to try this again if you want me to but I feel that the real time to fix this w ould be whenever we actually remove RefPath for netfx.

Reply:
I just don’t like this cruft in all our projects that area owners won’t understand.  Let’s make sure it gets cleaned up.